SPECS/xcp-ng-release.spec

```diff
@@ -355,47 +348,51 @@ if ! echo "$SSHD_PATCH" | patch --dry-run -RsN -d / -p1 >/dev/null; then
     fi
 fi
 
-%triggerin config -- openssh-clients
-# XCP-ng, 2023-01-31
-# We diverged here from xenserver-release-8.2.1-10: they chose to fully replace the sshd and ssh configuration each time either xenserver-release-config or openssh-client is updated.
-# We would like not to overwrite user configurations this way, so keeping the patches for XCP-ng 8.2.x
+%triggerin config -- openssh-clients >= 7.4p1-23.2.1
+# XCP-ng, 2024-05-13
+# Starting from openssh 7.4p1-23.2.1, the default ciphers, MACs, KexAlgorithms,
+# and HostKeyAlgorithms are now compiled in openssh code. This triggerin is used
+# to remove them from ssh_config. This is easier to do it through this package
+# rather than from the openssh one because this would be reverted if openssh is
+# upgraded without unpgrading xcp-ng-release.
 
 # # Replace openssh-clients config as openssh package mark it as noreplace as follows
 # # attr(0644,root,root) config(noreplace) {_sysconfdir}/ssh/ssh_config
 # install -D -m 644 %%{private_config_path}/ssh_config /etc/ssh/
 
+date >> /root/openssh-clients-triggerin
+echo $*   >> /root/openssh-clients-triggerin
+echo      >> /root/openssh-clients-triggerin
```

Comment:
What is this for? Testing?

Reply:
Indeed, a quick and ugly test :) Sorry for that one...